lib/safemode/core_jails.rb

```diff
@@ -300,75 +530,98 @@ def core_jail_class_methods(klass)
     sum
     swapcase
     swapcase!
-    toeuc
+    to_c
     to_f
     to_i
-    tojis
-    tosjis
+    to_r
     to_str
     to_sym
-    toutf16
-    toutf8
-    to_xs
```

Comment:
Bunch of other methods from http://ruby-doc.org/stdlib-2.3.6/libdoc/nkf/rdoc/String.html and I will add them later.